tests/test_explain_model.py

```diff
@@ -728,7 +728,7 @@ def test_explain_model_pandas_string(self, tabular_explainer):
         B = np.random.random(size=num_rows)
         C = np.random.randn(num_rows)
         cat = np.random.choice(['New York', 'San Francisco', 'Los Angeles',
-                                'Atlanta', 'Denver', 'Chicago', 'Miami', 'DC', 'Boston'], 100)
+                                'Atlanta', 'Denver', 'Chicago', 'Miami', 'DC', 'housing'], 100)
```

Comment:
this looks a bit weird/wrong?

Reply:
Reverted it. Thanks for pointing this out.